eq-author/src/App/page/Logic/BinaryExpressionEditor/NumberAnswerSelector.js

```diff
@@ -67,59 +79,81 @@ class NumberAnswerSelector extends React.Component {
     this.props.onConditionChange(value);
   };
 
+  handleError = () => {
+    const { expression } = this.props;
+
+    const { errorCode } = find(expression.validationErrorInfo.errors, {
+      errorCode: "ERR_NO_RIGHT_VALUE",
+    });
```

Comment:
This should use the constant that you've created rather than being hard coded.

Reply:
The constant is for the message not the errorcode passed in validationErrorInfo. Unless we add a constant.

Reply:
Nevermind, found it